python/ray/dashboard/client/src/common/DurationText/DurationText.tsx

```diff
@@ -27,7 +27,9 @@ export const DurationText = ({ startTime, endTime }: DurationTextProps) => {
 
   let durationText: string;
   let refreshInterval = 1000;
-  if (duration.asMinutes() < 1) {
+  if (duration.asSeconds() < 1) {
+    durationText = duration.format("SSS[ms]");
+  } else if (duration.asMinutes() < 1) {
```

Comment:
For numbers less than 1 minute, can we include milliseconds as well?

Reply:
also, can you add a test case in `DurationText.component.test.tsx`?

Reply:
> For numbers less than 1 minute, can we include milliseconds as well?

Sure, I can add that! I'll add it momentarily.

> also, can you add a test case in DurationText.component.test.tsx?

Yes, I'll take a look and see what I can come up with. Thanks for the review!